### ISSUE TITLE
PanelWWW

### ISSUE THREAD

**Issue by Mefisek** [lpiob/MTA-XyzzyRP#304]
Witam sciągnołem PHPbb3 i dodałem już dane do autoryzacja phpbb i nie wiem gdzie to wgrać może ktoś pomóc?

**Comment by Jaccck** [comment]
Tutaj znajdziesz odpowiedź - https://wiki.phpbb.com/Authentication_plugins

**Comment by Mefisek** [comment]
@Jaccck a jak wgrać ten admin panel na www: https://github.com/lpiob/MTA-XyzzyRP/tree/master/panelwww/administracja

**Comment by Jaccck** [comment]
Stronę umieść w osobnej domenie, np. http://adm.twojserwer.pl/

W pliku p/init.php wprowadź dane dostępowe do bazy danych i sól używaną przy tworzeniu kont użytkowników.

Dodatkowo w każdym pliku zamień przekierowania (adresy stron) na twoje. 

----

Pamiętaj że ten panel jest tylko przykładowy i nie należy się nim kierować. Najlepiej Ci będzie napisać własny w przyszłości. (lub przynajmniej przerobić go żeby jakoś wyglądał)

**Comment by Mefisek** [comment]
@Jaccck dobra a panel rejestracji ?

**Comment by Mefisek** [comment]
@Jaccck Wyszukaj we wszystkich plikach frazę "lss-rp" i popraw odwołania do adresów URL i ścieżek dostępu do IPBoard. mam tu rozumieć że muszę mieć IPB?

**Comment by Jaccck** [comment]
Nie, z tego co wiem panel rejestracji posiadał integracje z phpbb - odpowiadał za to plik https://github.com/lpiob/MTA-XyzzyRP/blob/master/panelwww/rejestracja_postaci/p/libs/phpbbauth.php

Niestety z panelem rejestracji postaci nie mam jak Ci pomóc, gdyż z niego nie korzystałem.

**Comment by Mefisek** [comment]
A to jak zrobić bo pobrałem IPB nulled https://github.com/lpiob/MTA-XyzzyRP/blob/master/opt/autoryzacja%20ipboard/loginauth_install.xml

**Comment by Mefisek** [comment]
Sory IP.Board 3.4.7 [Retail]

**Comment by Mefisek** [comment]
@GabWas Z tej wersji korzystam testowo bo szkoda mi używać na 5 min mojej licki.

**Comment by Jaccck** [comment]
Żeby wogóle wdrożyć autoryzacje IPB, musisz mieć wiedze na temat ipb i całego języka php.

Będziesz musiał przerobić bazę danych ipb do tego zainstalować bibliotekę adobd5 do php.
Lepiej już użyj darmowej wersji autoryzacji - phpBB niż używać nulled'ów, serio takiego nulled'a jest łatwo wykryć i to baardzo odrzuci graczy od Twojego serwera, nie lepiej na przykład użyć oprogramowania które jest darmowe i daje bardzo przybliżone możliwości? Na przykład właśnie phpBB lub MyBB.

Jakbyś chciał instalować autoryzacje graczy na phpBB, wystarczy ze wykonasz działania które napisałem Ci w moim pierwszym poście.

**Comment by Mefisek** [comment]
A jak by wyglądała sytuacja z MyBB ? Na tej samej zasadzie wgram jak w phpBB ?

**Comment by Jaccck** [comment]
Nie,phpBB i MyBB to dwa różne silniki forum z zapewne innym sposobem autoryzacji,  musiałbyś sam napisać lub poprosić/kupić od kogoś.

**Comment by luki2407** [comment]
@Mefisek aby użyć tej autoryzacji pod ipb musisz wgrać ją do public_html/admin/sources/loginauth/server   następnie instalujesz ręcznie w bazie. Kolumna "login_methods" dodajesz nową metode z pliku https://github.com/lpiob/MTA-XyzzyRP/blob/master/opt/autoryzacja%20ipboard/loginauth_install.xml next w panelu administracyjnym zmieniasz metode logowania ale pierw musisz zainstalować bibliotekę adobd5 do php. 

A pod myBB  tu masz gotowca :) https://mybboard.pl/watek-mybb-plugin-wykonujacy-sie-podczas-rejestacji_52637
